src/julia_gc.c

```diff
@@ -54,6 +54,8 @@
 // if MARKING_STRESS_TEST is defined, we stress test the TryMark code
 // #define MARKING_STRESS_TEST
 
+#define DISABLE_BIGVAL_TRACKING
```

Comment:
The correct way would be to uncomment the `#define` in line 43

Reply:
Will do.